packages/twenty-front/src/modules/settings/accounts/components/ImapConnectionForm.tsx

```diff
@@ -0,0 +1,143 @@
+
+const StyledFormContainer = styled.form`
+  display: flex;
+  flex-direction: column;
+  gap: ${({ theme }) => theme.spacing(4)};
+`;
+
+type ImapConnectionFormProps = {
+  control: Control<ImapConnectionFormValues>;
+  isEditing: boolean;
+};
+
+export const ImapConnectionForm = ({
+  control,
+  isEditing,
+}: ImapConnectionFormProps) => {
+  const { t } = useLingui();
+
+  return (
+    <Section>
+      <H2Title
+        title={t`IMAP Connection Details`}
+        description={
+          isEditing
+            ? t`Update your IMAP email account configuration`
+            : t`Configure your IMAP email account`
+        }
+      />
+      <StyledFormContainer>
+        <Controller
+          name="handle"
+          control={control}
+          render={({ field, fieldState }) => (
+            <TextInput
+              label={t`Email Address`}
+              placeholder={t`john.doe@example.com`}
+              value={field.value}
+              onChange={field.onChange}
+              error={fieldState.error?.message}
+            />
+          )}
+        />
+        <Controller
+          name="host"
+          control={control}
+          render={({ field, fieldState }) => (
+            <TextInput
+              label={t`IMAP Server`}
+              placeholder={t`imap.example.com`}
+              value={field.value}
+              onChange={field.onChange}
+              error={fieldState.error?.message}
+            />
+          )}
+        />
+        <Controller
+          name="port"
+          control={control}
+          render={({ field, fieldState }) => (
+            <TextInput
+              label={t`IMAP Port`}
+              type="number"
+              placeholder={t`993`}
+              value={field.value.toString()}
+              onChange={(value) => field.onChange(Number(value))}
+              error={fieldState.error?.message}
+            />
+          )}
+        />
+        <Controller
+          name="secure"
+          control={control}
+          render={({ field }) => (
+            <Select
+              label={t`Encryption`}
+              options={[
+                { label: 'SSL/TLS', value: true },
+                { label: 'None', value: false },
+              ]}
+              value={field.value}
+              onChange={field.onChange}
+              dropdownId="secure-dropdown"
+            />
+          )}
+        />
+        <Controller
+          name="password"
+          control={control}
+          render={({ field, fieldState }) => (
+            <TextInput
+              label={
+                isEditing
+                  ? t`Password (leave blank to keep current password)`
+                  : t`Password`
+              }
+              placeholder={t`••••••••`}
+              type="password"
+              value={field.value || ''}
+              onChange={field.onChange}
+              error={fieldState.error?.message}
+            />
+          )}
+        />
+        <Controller
```

Comment:
Let's keep a similar flow as others where the choice for {message visibility, contact creation settings etc} is differed to later